assets/js/base/components/payment-methods/style.scss

```diff
@@ -92,7 +93,8 @@
 
 	label {
 		@include reset-typography();
-		@include font-size(16px, 22px);
+		@include font-size(regular);
+		line-height: 1.375; // =22px when font-size is 16px.
```

Comment:
We originally had `line-height` included as an argument of the `font-size` mixin, why did you choose to extract it back out?

Reply:
The reason is similar to why I removed several calls to the `rem()` mixin. You can see [this comment](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2533#discussion_r429988538) for an example on how things might start getting difficult if we use px values when they are in fact `em` under-the-hood.

For the line-height, my reasoning was a bit the same. If the font sizes are named, does it make sense the line-height being in px? What if we change the `large` for size from `1.25` to `1.5` at some point? If line height are specified in px, we would need to update all of them while if they are unitless, they will automatically adapt.

I agree `1.375` looks like a magic number, here. That's why I added a comment next to it explaining where it comes from. Another option would be to create SCSS variables for different line heights so we can reuse them and we know where they come from. Do you think that would make it easier?